Src/Tracer.Logger.EurekaLog.pas

```diff
@@ -44,6 +44,17 @@ procedure TEurekaLogRefCountTracerLog.DoLogStackTrace;
 
 procedure TEurekaLogRefCountTracerLog.LogLine(
   const CallStack: TEurekaBaseStackList; const Index: Integer);
+
+  function GetAddressOf(var AVariable): string;
+  begin
+    Result := IntToHex( Integer( Pointer( @AVariable ) ), 8 );
+  end;
+
+  function PointerToHex(var AVariable): string;
```

Comment:
I didn't see you have this in unit Tracer.Logger.Tools.pas. Should be better now.